lerobot/common/robot_devices/motors/trossen_arm_driver.py

```diff
@@ -0,0 +1,264 @@
+import time
+import traceback
+import trossen_arm as trossen
+import numpy as np
+
+from lerobot.common.robot_devices.utils import RobotDeviceAlreadyConnectedError, RobotDeviceNotConnectedError
+from lerobot.common.robot_devices.motors.configs import TrossenArmDriverConfig
+
+import threading
+
+TROSSEN_ARM_MODELS = {
+    "V0_LEADER": [trossen.Model.wxai_v0, trossen.StandardEndEffector.wxai_v0_leader],
+    "V0_FOLLOWER": [trossen.Model.wxai_v0, trossen.StandardEndEffector.wxai_v0_follower],
+}
+
+class TrossenArmDriver:
+    """
+        The `TrossenArmDriver` class provides an interface for controlling 
+        Trossen Robotics' robotic arms. It leverages the trossen_arm_driver_py for communication with arms.
+
+        This class allows for configuration, torque management, and motion control of robotic arms. It includes features for handling connection states, moving the 
+        arm to specified poses, and logging timestamps for debugging and performance analysis.
+
+        ### Key Features:
+        - **Multi-motor Control:** Supports multiple motors connected to a bus.
+        - **Mode Switching:** Enables switching between position and gravity 
+        compensation modes.
+        - **Motion Interpolation:** Implements smooth motion using `PchipInterpolator` for trajectory generation.
```

Comment:
we use the driver to do the interpolation, we don't need to mention this anymore imo